src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -483,9 +484,80 @@ private void UpdateCustomColumnsVisibility(ImmutableDictionary<string, Immutable
                 }
             }
 
+            private void HideContainingMemberAndTypeColumns()
```

Comment:
Is this a clone of `UpdateCustomColumnsVisibility`? I think we want some better logic then special case it for hiding based on language name. Can you add this to the follow-up issue that you filed so we can clean it up?

Reply:
Also, are we planning to move these columns down to the platform layer? I think that is the best solution here, rather then us trying to special case things for specific Roslyn based languages.

Reply:
We talked offline and @chborl is going to add this to the tracking items to be addressed in the follow-up PR (which should happen soon)